### PR TITLE
dev-python/elasticsearch-py: bump to 2.4.0, drop old

### PR DESCRIPTION
**Pull request by hydrapolic**
Add new versions, drop old, drop mips.

Tests work without chroot (elasticsearch limitation).

### CODE REVIEW

File: dev-python/elasticsearch-py/elasticsearch-py-2.4.0.ebuild [hunk 1]
```diff
@@ -44,10 +48,10 @@ python_test() {
...
 	sed -i "s/# http.port: 9200/http.port: ${ES_PORT}/g; \
 	s/# cluster.name: my-application/cluster.name: gentoo-es-py-test/g" \
-	${ES}/config/elasticsearch.yml
+	${ES}/config/elasticsearch.yml || die
 
 	# start local instance of elasticsearch
-	${ES}/bin/elasticsearch -d -p ${PID}
+	${ES}/bin/elasticsearch -d -p ${PID} || die
 
 	for i in `seq 10`; do
```
Comment (SoapGentoo) [inline dev-python/elasticsearch-py/elasticsearch-py-2.4.0.ebuild]: declare `i` as local before

Comment (hydrapolic) [inline dev-python/elasticsearch-py/elasticsearch-py-2.4.0.ebuild]: Thanks, I forgot about this one when incorporating changes from es-curator.

**Comment by SoapGentoo** [comment]
@hydrapolic please squash the version bump and "local i" commit into one

**Comment by hydrapolic** [comment]
Thanks @SoapGentoo. I've read your comment too late, next time I'll try to squash the commits to avoid having too many commits.

**Pull request closed without merging**